src/ra.hrl

```diff
@@ -139,6 +155,7 @@
 -type snapshot_meta() :: #{index := ra_index(),
                            term := ra_term(),
                            cluster := ra_cluster_servers(),
+                           cluster_state => ra_cluster(),  %% TODO replace `cluster`
```

Comment:
would we really persist the whole `ra_cluster()` map in the snapshot? Only the voter_status is really required the rest of the fields are discovered during recovery etc.